doc/sdh/cli/cmd_install.adoc

```diff
@@ -102,22 +108,22 @@ syndesis install
 
 
 A route name can be given with `--route`.
-This step can be omitted as the operator is able to autodetect the route.
+This step can be omitted as the operator can autodetect the route.
 If you provide the route manually, you need to check your OpenShift installation.
 Typically the route name is the name of your OpenShift project followed by the cluster's hostname.
 E.g. a route `--route proj186023.6a63.fuse-ignite.openshiftapps.com` is specific to the Fuse Ignite test cluster `6a63.fuse-ignite.openshiftapps.com` and for the project `proj186023`.
-But you don't have to provide the route name.
+However, you don't have to provide the route name.
 
 If you want to have a link to the OpenShift console to read the Pod logs, you have to add the `--console` option with the full URL to the console.
-Unfortunately this cannot be autodetected.
-If not given, no link will appear.
+Unfortunately, this cannot be autodetected.
+If not given, no link appears.
 
 By default, this commands installs Syndesis in the currently connected project, but you can specify an alternative project with `--project <project>`.
 If this project already exists, it gets deleted unconditionally before the deployment, so be careful when using this option.
-By default you are asked whether you really want to delete the project for recreation.
+By default, you are asked whether you want to delete the project for recreation.
 You can switch off the security question with the option `--yes` (short: `-y`).
 
-WARNING: Don't use `syndesis install --project $(oc project -q) --yes`. You will shoot yourself into the foot. Ask the author if you want to know more details.
+WARNING: Don't use `syndesis install --project $(oc project -q) --yes`. You shoot yourself into the foot. Ask the author if you want to know more details.
```

Comment:
nit, grammar _You'll_ (I think)

Reply:
My grammarly says, that I should all write in present tense. But agreed, future makes more sense here ;-)

Reply:
I was always hesitant to install something that looks at all my text with no compensation from me (and it wasn't open source), that could be just my paranoia surfacing...